roles/common/tasks/main.yml

```diff
@@ -45,6 +45,7 @@
     - policycoreutils-python
     - unzip
     - jq
+    - java-1.8.0-openjdk
```

Comment:
We do need it to be present on every single worker node since elasticsearch is running as a framework and can end up with a node on any worker. However, it is not strictly needed on control or edge nodes.